### PR TITLE
Fix hk and lttp docs

### PR DESCRIPTION
**Pull request by Ijwu**

### CODE REVIEW

**Comment by Ijwu** [comment]
@Berserker66 I put some TODOs in that require your input as I am still somewhat ignorant of how things tie together.

File: Patch.py [hunk 1]
```diff
@@ -304,7 +304,7 @@ def read_rom(stream, strip_header=True) -> bytearray:
                     print(f"Creating patch for {rom}")
                     result = pool.submit(create_patch_file, rom, address)
                     result.add_done_callback(lambda task: print(f"Created patch {task.result()}"))
-
+                # TODO: @berserker does this if tree require updating to refer to new extensions?
```
Comment (Berserker66) [inline Patch.py:307]: It *could* be added. Though running Patch.py directly has kind of fallen out of general use, we don't even freeze it for users.

File: LttPAdjuster.py [hunk 2]
```diff
@@ -185,6 +185,7 @@ def adjustGUI():
     romVar2 = StringVar()
     romEntry2 = Entry(romDialogFrame, textvariable=romVar2)
 
+    # TODO: @berserker does this need an update to handle .aplttp?
```
Comment (Berserker66) [inline LttPAdjuster.py:188]: yes

File: Patch.py (hunk 1 above)
Comment (Ijwu) [inline Patch.py:307]:
```suggestion
```

File: LttPAdjuster.py (hunk 2 above)
Comment (Ijwu) [inline LttPAdjuster.py:188]:
```suggestion
```

**Pull request closed without merging**